### ISSUE TITLE
Review by Maxiausder4a 

### ISSUE THREAD

**Issue by Maxiausder4a** [LTK4MI/Homework_OSCEM#5]
Content of the Repository: 

The Repositorys includes the homework assignment of the first homework of the course Open Source Energy System Modeling, It contains the necessary files: LICENSE, README, code.py and test_code.py.  The Github Actions contain the linter Ruff and pytest.yaml. Moreover, it contains an approved open source license (Apache License 2.0).

- The Pull Request Update README did not activate the Github Action pytest.yaml, and is now awaiting approval from the maintainer.
- The Pull Request Update README did not activate the Linter ruff.yaml , and is now awaiting approval from the maintainer.

- The Pull Request Testint the Linter Ruff by implementing Indentation Errors in code.py did not activate the the workflows pytest, and ruff, since it is awaiting approval by the maintainer. 

=> It seems, that testing the workflows is not possible for me, since they are awainting approval from the maintainer to start testing.

**Comment by Maxiausder4a** [comment]
Please review @danielhuppmann

**Comment by Maxiausder4a** [comment]
After some time, all tests have failed as expected.

**Comment by danielhuppmann** [comment]
Correct, a repository-admin (in this case me) had to approve the GitHub Actions before they are started.
